crates/zoon/src/element/raw_el.rs

```diff
@@ -76,6 +77,10 @@ pub trait RawEl: Sized {
         self.attr("id", &id.into_cow_str())
     }
 
+    fn lang<'a>(self, lang: Lang) -> Self {
+        self.attr("lang", &lang.as_str())
```

Comment:
I think the `&` is no longer needed here. The compiler is able to automatically dereference double references but we don't need it here. I think Clippy can prevent/check such things automatically.